pkg/controllers/repositoryserver/suite_test.go

```diff
@@ -15,58 +15,100 @@
 package repositoryserver
 
 import (
-	"path/filepath"
+	"context"
 	"testing"
-	"time"
 
 	. "gopkg.in/check.v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/kubernetes/scheme"
-	"sigs.k8s.io/controller-runtime/pkg/client"
-	"sigs.k8s.io/controller-runtime/pkg/envtest"
+	ctrl "sigs.k8s.io/controller-runtime"
 
-	crkanisteriov1alpha1 "github.com/kanisterio/kanister/pkg/apis/cr/v1alpha1"
+	crclientv1alpha1 "github.com/kanisterio/kanister/pkg/client/clientset/versioned/typed/cr/v1alpha1"
+	"github.com/kanisterio/kanister/pkg/kube"
+	"github.com/kanisterio/kanister/pkg/resource"
 )
 
 // Hook up gocheck into the "go test" runner.
 func Test(t *testing.T) { TestingT(t) }
 
-type ControllerSuite struct {
-	testEnv *envtest.Environment
+type RepoServerControllerSuite struct {
+	crCli                         crclientv1alpha1.CrV1alpha1Interface
```

Comment:
We can remove it if we are not using it anywhere.
```suggestion
```

Reply:
This is used in the next PR in the chain of these PRs
https://github.com/kanisterio/kanister/pull/1997